### PR TITLE
feat: makefile support

### LINKED ISSUE #920: Makefile
### Language Name 🖊

A simple Makefile only project is not detected

### Logo 📷

gnu?
```
    _-`````-,           ,- '- .
  .'   .- - |          | - -.  `.
 /.'  /                     `.   \
:/   :      _...   ..._      ``   :
::   :     /._ .`:'_.._\.    ||   :
::    `._ ./  ,`  :    \ . _.''   .
`:.      /   |  -.  \-. \\_      /
  \:._ _/  .'   .@)  \@) ` `\ ,.'
     _/,--'       .- .\,-.`--`.
       ,'/''     (( \ `  )    
        /'/'  \    `-'  (      
         '/''  `._,-----'
          ''/'    .,---'
           ''/'      ;:
             ''/''  ''/
               ''/''/''
                 '/'/'
                  `;
```

Or 

Reuse/modify shell scripts logo?

### Upstream support ✅

- [X] Does tokei already support the language in question?

### PR DESCRIPTION
**Pull request by ozwaldorf**
# What

Adds makefile support

# Demo

image

# Credits

ASCII was found online at https://www.gnu.org/graphics/alternative-ascii.html under GPLv2

# Related

closes #920

### CODE REVIEW

File: languages.yaml [hunk 1]
```diff
@@ -1427,6 +1427,37 @@ Lua:
...
+  colors:
+    ansi:
+      - white
+      - yellow
+      - red
+    hex:
+      - '#FFFFFF'
+      - '#FAEC9A'
+      - '#610000'
+    chip: '#610000'
```
Comment (o2sh) [inline languages.yaml:1460]:
```suggestion
    chip: '#427819'
```

cf. linguist entry for [Makefile](https://github.com/github/linguist/blob/c1c34e5260797b4d598f5ec76f19723bfc5a1894/lib/linguist/languages.yml#L3833)

**Review by o2sh — APPROVED** [review]
The ASCII logo looks really nice! 
I left a remark concerning the chip color.

Thanks a lot for your contribution @ozwaldorf (100th language 🎊 🎊 )

File: languages.yaml (hunk 1 above)
Comment (ozwaldorf) [reply languages.yaml]: Commited https://github.com/o2sh/onefetch/pull/921/commits/5bf867ffb59b9e662c627ff46019a6bf4e15eaeb thanks for the suggestion!

**Comment by ozwaldorf** [comment]
@o2sh thank *you* for an amazing cli tool! Happy to contribute :heart: 

I amended the chip color and should be good to go now!

**Pull request merged**